src/cc_catalog_airflow/dags/provider_api_scripts/cleveland_museum_of_art.py

```diff
@@ -0,0 +1,213 @@
+def handle_the_response(response):
+    for data in response:
+        foreign_url = data.get('url', None)
+        single_image = data.get('images')
+        image_url = None
+        images_till_now = 0
+        if single_image is not None:
+            if single_image.get('web'):
+                key = 'web'
+                image_url = single_image.get('web').get('url', None)
+            elif single_image.get('print'):
+                key = 'print'
+                image_url = single_image.get('print').get('url', None)
+            else:
+                key = 'full'
+                image_url = single_image.get('full').get('url', None)
+            
+            width = single_image[key]['width']
+            height = single_image[key]['height']
+            foreign_id = data.get('id', image_url)
+            license_status = data.get('share_license_status', None).lower()
+            license_share = license_status.lower()
+            title = data.get('title', '')
+
+            creator_info = data.get('creators', {})
+            creator_name = None
+            if creator_info:
+                creator_name = creator_info[0].get('description', '')
+            meta_data = create_meta_data(data)
+            images_till_now = image_store.add_item(
+                foreign_url,  # Foreign Landing URl
+                image_url,  # Image URL
+                None,  # Thumbnail_URL
+                None,  # License_URL
+                license_status,  # License
+                "1.0",  # License_Version
+                foreign_id,  # Foreign Identifier
+                width,  # Width of the image
+                height,  # height of the image
+                creator_name,  # Creator Name
+                None,  # Creator URl
+                title,  # Title of the Image
+                meta_data,  # Meta Data
+                None,  # Raw Tags
+                'f',  # Watermarked
+                None,  # Source
+            )
+    return images_till_now
+
+
+
+def create_meta_data(data):
+    meta_data = {}
+    meta_data['accession_number'] = data.get('accession_number', '')
+    meta_data['technique'] = data.get('technique', '')
+    meta_data['date'] = data.get('date', '')
+    meta_data['credit_line'] = data.get('creditline', '')
+    meta_data['classificatoin'] = data.get('type', '')
+    meta_data['culture'] = ','.join(list(filter(
+        None, data.get('culture', ''))))
+    meta_data['tombstone'] = data.get('tombstone', '')
+    return meta_data
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(
+        description='Cleveland Museum of Art API',
+        add_help=True
+    )
+    parser.add_argument(
+        '--date',
+        help='Identify the images uploaded on a date(YY-MM-DD)'
+    )
+    args = parser.parse_args()
+    if args.date:
+        date = args.date
+    else:
+        date_obj = datetime.now() - timedelta(days=2)
+        date = datetime.strftime(date_obj, '%Y-%m-%d')
+
+    main(date)
```

Comment:
None of the `date` related logic is necessary (again, see the updated issue).  This can simplified to:
```python
if __name__ == '__main__':
    main()
```